### DISCUSSION TITLE
239

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#164] [Comments]
# 239

https://linuxize.com/post/how-to-enable-ssh-on-ubuntu-20-04/ ​ Ubuntu 를 설치 했다면 상황에 따라 외부에서의 SSH 를 허용 해 두어야 할 경우가 있습니다. 보통의 경우 ubuntu 를 설치 했을 때 openssh 서버가 설치 되어 있지 않기 때문에 설치를 해 주어야 합니다. 차근차근 현재 사용중인 Ubuntu 디바이스의 SSH 접속을 허용해 보도록 하겠습니다. 참고로 외부 ip 에서의 접속까지 허용을 하려면 공유기에서 22번 port를 port forwarding 해주어야 합니다. iptime 을 사용 한다면 192.168.0.1 로 접속해서 특정 로컬아이피의 포트를 포트 포워딩 해 주면 됩니다. 전제조건 반드시 sudo 권한으로 로그인 해야 합니..

https://shanepark.tistory.com/239

**Comment by tiamoirony** [comment]
좋은글 감사합니다. 
코드 블럭은 어떻게 만든신 건가요 ?
글쓰기 - 코드불럭 이런식 말고 다른 방법이 있나요 ?

**Reply by ShanePark** [reply]
안녕하세요. 제가 인텔리제이 IDE를 좋아해서, 그 디자인을 본따서 css를 편집해 만들었어요
